src/Features/Core/Portable/FindUsages/DefinitionItem.DocumentLocationDefinitionItem.cs

```diff
@@ -37,34 +37,33 @@ internal sealed class DefaultDefinitionItem : DefinitionItem
             {
             }
 
+            private DocumentSpan? TryGetDocumentSpan(Workspace workspace)
+            {
+                var currentSolution = workspace.CurrentSolution;
+                var document = currentSolution.GetDocument(SourceSpans[0].DocumentId);
```

Comment:
📝 This differs from `SerializableDocumentSpan.RehydrateAsync` (it doesn't have the fallback call to `GetSourceGeneratedDocumentAsync`)

Reply:
➡️ Now includes the fallback